### DISCUSSION TITLE
Adding video to instructions

### DISCUSSION THREAD

**Discussion by fspinar** [jspsych/jsPsych#2667] [Q&A] [answered]
Hey all,

I want to add a video to my instructions while using the instructions plugin. Any ideas on how to do that? I managed to add images as html strings but not videos.

Thanks a lot.

**Comment by jodeleeuw** [comment, marked as answer]
Hi @fspinar,

You should be able to embed a `<video>` element in the same way that you added the images. Let us know if that doesn't work for some reason!

**Reply by fspinar** [reply]
Do you mean like this @jodeleeuw 

`<video src="static/videos/choice.mp4"></video>`

This way, for some reason I only see the initial picture of the video and it doesn't move at all.

**Reply by jodeleeuw** [reply]
You can add the controls and/or auto play attribute to make it playable. Here's a useful reference page: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/video

**Reply by fspinar** [reply]
Thanks a lot! It worked fine.
